docs/resources/generated-error-pages/error-codes-inventory.rst.inc

```diff
@@ -492,13 +492,163 @@ PACKAGE_UPLOAD_REJECTED
+
+
+
+
+.. _error_code_PARTY_NOT_FOUND:
+
+PARTY_NOT_FOUND
+---------------------------------------------------------------------------------------------------------------------------------------
+    
+    **Explanation**: The party referred to by the request was not found.
+
+    **Category**: InvalidGivenCurrentSystemStateResourceMissing
+
+    **Conveyance**: This error is logged with log-level INFO on the server side. This error is exposed on the API with grpc-status NOT_FOUND including a detailed error message
+
+    **Resolution**: Check that you are connecting to the right participant node and the party is spelled correctly.
+
+
+
+2.2.1.2. ParticipantErrorGroup / LedgerApiErrors / AdminServices / UserManagementServiceErrors
+===================================================================================================================
+
+
+
+
+
+.. _error_code_CONCURRENT_USER_UPDATE_DETECTED:
+
+CONCURRENT_USER_UPDATE_DETECTED
+---------------------------------------------------------------------------------------------------------------------------------------
+    
+    **Explanation**: Concurrent updates to a user can be controlled by optionally supplying an update request with a resource version. A user's resource version can be obtained by reading the user on the Ledger API. There was attempt to update a user using a stale resource version indicating a different processes had updated the user earlier.
+
+    **Category**: ContentionOnSharedResources
+
+    **Conveyance**: This error is logged with log-level INFO on the server side. This error is exposed on the API with grpc-status ABORTED including a detailed error message
+
+    **Resolution**: Restart the user update procedure by reading this user again to obtain its most recent state and in particular its most recent resource version.
+
+
+
+
+.. _error_code_INVALID_USER_UPDATE_REQUEST:
+
+INVALID_USER_UPDATE_REQUEST
+---------------------------------------------------------------------------------------------------------------------------------------
+    
+    **Explanation**: There was an attempt to update a user using an invalid update request.
+
+    **Category**: InvalidIndependentOfSystemState
+
+    **Conveyance**: This error is logged with log-level INFO on the server side. This error is exposed on the API with grpc-status INVALID_ARGUMENT including a detailed error message
+
+    **Resolution**: Inspect the error details for specific information on what made the request invalid. Retry with an adjusted update request.
+
+
+
+
+.. _error_code_MAX_USER_ANNOTATIONS_SIZE_EXCEEDED:
+
+MAX_USER_ANNOTATIONS_SIZE_EXCEEDED
+---------------------------------------------------------------------------------------------------------------------------------------
+    
+    **Explanation**: A user can have only a limited amount of annotations. There was an attempt to create or update a user with too many annotations.
```

Comment:
```suggestion
    **Explanation**: A user can have only a limited number of annotations. There was an attempt to create or update a user with too many annotations.
```

Reply:
Similar to `MAX_PARTY_DETAILS_ANNOTATIONS_SIZE_EXCEEDED` I'm rewording this to state:
> A user can have at most 256kb worth of annotations in total measured in number of bytes in UTF-8 encoding.
There was an attempt to create or update a user such that this limit would have been exceeded.